hz: 10 #ループ周期
goal_index: 50 #グローバルパス内のローカルゴールのインデックス
local_goal_dist: 1 #ゴールまでの距離
